superset/assets/src/dashboard/components/Dashboard.jsx

```diff
@@ -339,6 +341,7 @@ class Dashboard extends React.PureComponent {
           addFilter={this.modalAddFilterHandler}
           modalTitle={this.modalTitle}
           datasource={this.modalDatasource}
+          dashboardState={this.props.dashboardState}
```

Comment:
Now, we can remove dashboardState props..

Reply:
Good catch! Done